sql/rename.go

```diff
@@ -209,3 +209,78 @@ func (p *planner) RenameIndex(n *parser.RenameIndex) (planNode, error) {
 
 	return &valuesNode{}, nil
 }
+
+// RenameColumn renames the column.
+// Privileges: CREATE on table.
```

Comment:
yup. Anything modifying the schema requires the same privileges as creating it (although mysql also has a `ALTER` privilege, but we don't).